### ISSUE TITLE
Container image does not support IPv6-first Kubernetes environments

### ISSUE THREAD

**Issue by centromere** [percona/pmm#2709]
### Description

By default, the nginx configuration looks like this:
```
  server {
    listen 80;
    listen 443 ssl http2;
```
which results in sockets looking like this:
```
# ss -lntp | grep nginx
LISTEN 0      511          0.0.0.0:443        0.0.0.0:*    users:(("nginx",pid=743682,fd=7),("nginx",pid=743681,fd=7),("nginx",pid=743566,fd=7))
LISTEN 0      511          0.0.0.0:80         0.0.0.0:*    users:(("nginx",pid=743682,fd=6),("nginx",pid=743681,fd=6),("nginx",pid=743566,fd=6))
#
```

This is not good, because in IPv6-first Kubernetes clusters, the kubelet is going to try to assess Pod readiness by connecting to its IPv6 address. In the current state, the kubelet will receive a `connection refused` error, and the Pod will never become `Ready`.

### Expected Results

I expect that the Pod achieves `Ready` status without having to manually modify the `pmm.conf` nginx configuration file.

### Actual Results

nginx listens on `0.0.0.0` which causes the readiness check to fail.

### Version

PMM server 2.40.1

### Steps to reproduce

Install the `percona/pmm` chart on a Kubernetes cluster which uses IPv6-first and observe that the Pod is never marked as `Ready`.

### Code of Conduct

- [X] I agree to follow Percona Community Code of Conduct

**Comment by ademidoff** [comment]
@centromere Thanks for raising this, it indeed looks like an issue!

Do you want to suggest a fix? Feel free to make PR if you do - that may speed things up :)

**Comment by ademidoff** [comment]
I wonder if something like this would work for everybody without trouble:
```
  server {
    listen 80;
    listen 443 ssl http2;
    listen [::]:80;
    listen [::]:443 ssl http2;
    # ...
  }
```

**Comment by AndreKR** [comment]
Yes, I think the missing `listen [::]:8080` and `listen [::]:8443` lines should be added.

(In fact [I think that nginx shouldn't require them](https://github.com/nginx/nginx/issues/444), but unfortunately that's the way it is.)

Theoretically `entrypoint.sh` could do some checking, like whether `/proc/net/if_inet6` exists or something and only then enable the extra `listen` lines via environment variable, but I don't think there is any realistic environment where this would be necessary. Remember, the system doesn't need to have a working IPv6 address, it only needs a kernel that can do IPv6, which I think can be universally expected nowadays.
